vignettes/intro-to-hector.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: A tour of the Hector R interface
+author: Alexey Shiklomanov
+---
+
+This vignette provides a basic introduction for running Hector with R.
+First, it shows how to do a simple Hector run with one of Hector's built-in scenarios.
+Then, it shows how to modify Hector parameters from within R and uses this functionality to perform a simple sensitivity analysis of several Hector variables to the CO$_2$ fertilization parameter $\beta$.
```

Comment:
Might be clearer to say "sensitivity analysis of how the CO2 fertilization parameter beta affect several Hector output variables."